packages/runtime/id-compressor/src/types/identifiers.ts

```diff
@@ -24,7 +24,7 @@ export type OpSpaceCompressedId = number & {
 };
 
 /**
- * A version 4, variant 1 uuid (https://datatracker.ietf.org/doc/html/rfc4122).
+ * A {@link https://datatracker.ietf.org/doc/html/rfc4122 | version 4, variant 1 uuid}.
```

Comment:
Changed this because in the previous approach, the closing parenthesis was being considered part of the link, rendering a final URL with a `%29` (encoded closing parenthesis) at the end that doesn't work. Visible today in [the published docs](https://fluidframework.com/docs/api/v2/id-compressor/stableid-typealias) (note the link target as I hovered over it):
![image](https://github.com/microsoft/FluidFramework/assets/716334/1e8e395d-7bf7-4d30-bdd7-7f6e2fb41b26)

Confirmed locally that the new approach renders the link correctly.

Reply:
Regardless of broken-ness, the way you did it is just nicer anyway. Thanks!